components/list/test/list.visual-diff.js

```diff
@@ -147,8 +148,14 @@ describe('d2l-list', () => {
 			{ name: 'some selected', selector: '#selectableSomeSelectedControls' },
 			{ name: 'all selected', selector: '#selectableAllSelectedControls' },
 			{ name: 'all selected pages', selector: '#selectableAllSelectedControlsPages' },
-			{ name: 'sticky top', selector: '#stickyControls', action: () => scrollTo('#stickyControls > div', 0) },
-			{ name: 'sticky scrolled', selector: '#stickyControls', action: () => scrollTo('#stickyControls > div', 45) }
+			{ name: 'sticky top', selector: '#stickyControls', action: async() => {
+				await scrollTo('#stickyControls > div', 45);
+				await scrollTo('#stickyControls > div', 0);
```

Comment:
I'm pretty confused about why this would help... are you sure this isn't essentially just increasing the timeout so that the `scrollIntoView` has properly triggered? The number only changes the vertical scrolling within the div, and I really can't see the browser getting confused about that part.

Did you try this same fix with `0` both times?

Reply:
Thanks Stacey! We just caught up, Stacey showed me these failures in action. Scrolling the container down and back up definitely fixes them - which tells me there might be something wrong in the sticky logic itself! I'll have a look and see if I can find out why.